docs/01-get-started/your-first-route.mdx

```diff
@@ -6,12 +6,48 @@ sidebar_position: 3
 
 <p>{frontMatter.description}</p>
 
-:::info WIP
+# Routes
 
-Add instructions here:
+Starting v3.0.0, Front-Commerce uses Remix's
+[API Routes](https://remix.run/docs/en/main/guides/api-routes) to handle all the
+routing of you application. All routes are located under `app/routes` folder.
 
-- Create a route
-- Introduce loader
-- Redirect to Remix guides
+API Routes will use the default export of each Route file to generate the
+route's UI. Go ahead and create a new route `hello-world.tsx`:
 
-:::
+```tsx title="app/routes/hello-world.tsx"
+export default function HelloWorld() {
+  return (
+    <div>
+      <h1>Hello world!</h1>
+    </div>
+  );
+}
+```
+
+# Loaders
+
+In order to display dynamic content in your routes, you will need to fetch some
+data. This is done by using Remix's Routes API method `loader`.
+
+Here is a basic example of how a `loader` could be used:
+
+```tsx title="app/routes/hello-world.tsx"
+export const loader = () => {
```

Comment:
**issue:** we should take the habit of always adding typings and use TS.

**suggestion:** type the loader and the `useLoaderData` (with `@remix-run`'s import for now because in this example we don't yet do anything specific to FC.